docs/machine-learning/tutorials/sentiment-analysis.md

```diff
@@ -105,26 +107,29 @@ Classification algorithms are frequently one of the following types:
 
 ### Prepare your data
 
-1. Download the [Wikipedia detox-250-line-data.tsv](https://github.com/dotnet/machinelearning/blob/master/test/data/wikipedia-detox-250-line-data.tsv) and the [wikipedia-detox-250-line-test.tsv](https://github.com/dotnet/machinelearning/blob/master/test/data/wikipedia-detox-250-line-test.tsv) data sets and save them to the *Data* folder previously created. The first dataset trains the machine learning model and the second can be used to evaluate how accurate your model is.
+1. Download [The UCI Sentiment Labeled Sentences dataset zip file (see citations in the following note)](https://archive.ics.uci.edu/ml/machine-learning-databases/00331/sentiment%20labelled%20sentences.zip), and unzip.
+
+2. Copy the `yelp_labeled.txt` file into the *Data* directory you created.
```

Comment:
Change file name to match the name in the code sample -> yelp_labelled.txt

Reply:
Good catch.